mailgreen/controller/mail_controller.py

```
@@ -163,3 +182,96 @@ async def get_sender_details(
+
+class DeleteMailsRequest(BaseModel):
+    message_ids: List[str]
+    confirm: bool = False  # false면 삭제하지 않고 추정만
+
+
+@router.delete("/trash")
+async def trash_mails(
+    user_id: str = Query(..., description="User UUID"),
+    payload: DeleteMailsRequest = Body(
+        ..., description="삭제할 메일 ID 목록과 확인 여부"
+    ),
```

Ruff (0.11.9)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)